4.1/start.sh

```diff
@@ -20,6 +25,7 @@ if [ ! -f ${IMPORT_FINISHED} ]; then
   /app/init.sh
   touch ${IMPORT_FINISHED}
 else
+  rm -f /etc/postgresql/14/main/conf.d/postgres-import.conf
```

Comment:
this file is already removed after the import, see: https://github.com/mediagis/nominatim-docker/blob/322deccac655584add9c4db08985323f506ee30f/4.1/init.sh#L107

any other reasons why you have added this here again?

Reply:
this file is not in any persistent place. after a recreation of the container (`docker-compose down; docker-compose up`) or after an update of the image, the state is reset and this file is there again.

Reply:
another approach would be to only copy the import-optimizations to the conf.d folder before the init and remove it again afterwards, that way it is not in the image and does not need to be removed at every start.

Reply:
I like this approach.

Reply:
from the postgresql.conf file:

```
# These options allow settings to be loaded from files other than the
# default postgresql.conf.  Note that these are directives, not variable
# assignments, so they can usefully be given more than once.

include_dir = 'conf.d'			# include files ending in '.conf' from
					# a directory, e.g., 'conf.d'
```

so the file could be copied to `/etc/postgresql/14/main/conf.d/postgres-import.conf.disabled`
and renamed for the import.

should I change the Dockerfile and scripts to do it that way?

Reply:
Sounds like a good solution but to honest I don't have a very strong opinion how the file should be activated and deactivated.

Reply:
@leonardehrenfried I just updated the dockerfile and scripts to just enable the unsafe overrides during the import.

PS: I also shared my server setup info on #265